modules/core/core-api/src/main/java/com/enonic/xp/content/ModifiableContent.java

```diff
@@ -0,0 +1,161 @@
+package com.enonic.xp.content;
+
+
+import java.time.Instant;
+import java.util.EnumSet;
+import java.util.Locale;
+import java.util.function.Function;
+
+import com.enonic.xp.annotation.PublicApi;
+import com.enonic.xp.attachment.Attachments;
+import com.enonic.xp.data.PropertyTree;
+import com.enonic.xp.icon.Thumbnail;
+import com.enonic.xp.page.Page;
+import com.enonic.xp.security.PrincipalKey;
+
+@PublicApi
+public class ModifiableContent
+{
+    public final Content source;
+
+    public EditableFieldPolicyWrapper<String> displayName;
+
+    public EditableFieldPolicyWrapper<PropertyTree> data;
+
+    public EditableFieldPolicyWrapper<ExtraDatas> extraDatas;
+
+    public EditableFieldPolicyWrapper<Page> page;
+
+    public EditableFieldPolicyWrapper<Boolean> valid;
+
+    public EditableFieldPolicyWrapper<Thumbnail> thumbnail;
+
+    public EditableFieldPolicyWrapper<PrincipalKey> owner;
+
+    public EditableFieldPolicyWrapper<Locale> language;
+
+    public EditableFieldPolicyWrapper<PrincipalKey> creator;
+
+    public EditableFieldPolicyWrapper<Instant> createdTime;
+
+    public EditableFieldPolicyWrapper<PrincipalKey> modifier;
+
+    public EditableFieldPolicyWrapper<Instant> modifiedTime;
+
+    public EditableFieldPolicyWrapper<ContentPublishInfo> publishInfo;
+
+    public EditableFieldPolicyWrapper<ContentIds.Builder> processedReferences;
```

Comment:
this is the only builder (although I see some mutable types). what is the point?